test/common/gristUrls.ts

```diff
@@ -64,4 +64,44 @@ describe('gristUrls', function() {
       assert.deepEqual(parseFirstUrlPart('o', ''), {path: ''});
     });
   });
+
+  describe('getHostType', function() {
+    const defaultOptions = {
+      baseDomain: 'getgrist.com',
+      pluginUrl: 'https://plugin.getgrist.com/path',
+    };
+    afterEach(function () {
+      delete process.env.APP_DOC_INTERNAL_URL;
```

Comment:
Take a look in `DocApi2.ts` tests, we have an utility class to work with env variables `new testUtils.EnvironmentSnapshot();`

Reply:
Thanks! I took into account all your remarks in this commit: 3aa126133fea7dc372c229c0f3b06f351cb6cd40